contracts/test/contract.OperatorRegistry.test.js

```diff
@@ -113,7 +101,9 @@ contract("OperatorRegistry", (accounts) => {
         verifiedShareOperatorEOAs[i]
       );
 
-      const fundsDelta = newBalance - initialBalance[i];
+      const fundsDelta = web3.utils
```

Comment:
@un7c0rn I was trying to make changes based on this comment, but my local truffle test is failing as below 
![image](https://github.com/user-attachments/assets/a469ca6c-9c1e-45c6-a47d-4591e4532004)
Could you please let me know how to resolve this? I pulled latest code from this branch.thanks

Reply:
@xiangformless Does it pass if we restore this file to main?

Reply:
after reverting back to main, this file is still getting the same error. as below. 
![image](https://github.com/user-attachments/assets/4d348bd1-4ec8-4f17-bced-11f9c1464d5f)
I am curious how you get these tests passed successfully earlier. As I am only making minor changes, I do not think these changes can causes these failures.

Reply:
@xiangformless I think you removed the optimization changes in this PR when I asked if they were required, perhaps that's what's causing the issue here? The error is that the final binary compilation of the contract is too big for the chain. Optimization should help with this.

Reply:
@un7c0rn Yes, I added those optimization for debugging purpose. I thought it might be redundant after your comments. Let me put them back to see if it can pass. thanks

Reply:
Ok taking a look @xiangformless!

Reply:
@xiangformless I've updated the truffle config and am re-kicking the tests now.